palace/linalg/superlu.hpp

```diff
@@ -41,7 +41,22 @@ class SuperLUSolver : public mfem::Solver
   // Sets matrix associated with the SuperLU solver.
   void SetOperator(const mfem::Operator &op) override;
 
+  // Application of the solver.
   void Mult(const mfem::Vector &x, mfem::Vector &y) const override { solver.Mult(x, y); }
+  void ArrayMult(const mfem::Array<const mfem::Vector *> &X,
+                 mfem::Array<mfem::Vector *> &Y) const override
+  {
+    solver.ArrayMult(X, Y);
+  }
+  void MultTranspose(const mfem::Vector &x, mfem::Vector &y) const override
+  {
+    solver.MultTranspose(x, y);
+  }
+  void ArrayMultTranspose(const mfem::Array<const mfem::Vector *> &X,
+                          mfem::Array<mfem::Vector *> &Y) const override
+  {
+    solver.ArrayMultTranspose(X, Y);
+  }
```

Comment:
If we're not using the methods in any of the compilation, suggests we don't even really need them, or at least that mfem apparently doesn't deem completing them to be necessary for implementing an operator.

Reply:
The methods have (unoptimized) implementations in the base `mfem::Operator` class.